instancio-core/src/main/java/org/instancio/internal/beanvalidation/JavaxAndJakartaBeanValidationHandlerResolver.java

```diff
@@ -107,33 +127,30 @@ public void process(final Annotation annotation,
                             final GeneratorSpec<?> spec,
                             final Field field,
                             final Class<?> fieldType) {
-
-            final Digits digits = (Digits) annotation;
+            final int integer = ReflectionUtils.getAnnotationParameter(annotation, "integer");
```

Comment:
I'm also not sure about using reflection to read the annotation values. It may seem odd for a library that relies on reflection to populate objects :) However, I think we should avoid reflection on JDK/library classes whenever possible. There may also be some implications in terms of JPMS.

I think making `FieldAnnotationHandler` implementations reusable between `javax` and `jakarta` would be a better approach.

Reply:
Ok, If You think that reflection should not used for this case I will remove it